src/avatar.jsx

```diff
@@ -1,8 +1,45 @@
 import React from 'react';
-import StylePropable from './mixins/style-propable';
 import Colors from './styles/colors';
 import getMuiTheme from './styles/getMuiTheme';
 
+function getStyles(props, state) {
+  const {
+    backgroundColor,
+    color,
+    size,
+    src,
+  } = props;
+
+  const {
+    avatar,
+  } = state.muiTheme;
+
+  const styles = {
+    root: Object.assign({
```

Comment:
no worries, programming is a playful job. It always tickles you to write clever code. We've all been there :laughing: :laughing: